### PR TITLE
Temporarily skip ExecGraphTests.CheckExecGraphInfoSerialization on GPU

### PR DESCRIPTION
**Pull request by ababushk**

### CODE REVIEW

File: inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/keep_assing.cpp [hunk 1]
```diff
@@ -19,6 +19,8 @@ std::string ExecGraphKeepAssignNode::getTestCaseName(testing::TestParamInfo<std:
  * So exec graph may lose it. Will check that it's present in dumped exec graph.
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/keep_assing.cpp:19]: `#include "functional_test_utils/skip_tests_config.hpp"`

File: inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/num_inputs_fusing_bin_conv.cpp [hunk 2]
```diff
@@ -50,6 +50,8 @@ void ExecGraphInputsFusingBinConv::TearDown() {
 }
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/num_inputs_fusing_bin_conv.cpp:50]: `#include "functional_test_utils/skip_tests_config.hpp"`

File: inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/unique_node_names.cpp [hunk 3]
```diff
@@ -60,6 +60,8 @@ void ExecGraphUniqueNodeNames::TearDown() {
 }
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/unique_node_names.cpp:60]: `#include "functional_test_utils/skip_tests_config.hpp"`

**Review by iefode — APPROVED** [review]

**Comment by ilyachur** [comment]
@jdanieck FYI

**Pull request merged**